modules/utxo-lib/src/bitgo/parseInput.ts

```diff
@@ -403,6 +404,7 @@ const parseP2tr2Of3: InputParser<ParsedSignatureScriptTaproot> = (p) => {
     signatures: match[':signature'] as [Buffer, Buffer],
     controlBlock,
     scriptPathLevel,
+    leafVersion: controlBlock[0] & 0xfe,
```

Comment:
Might be nice to add a citation as to where this comes from, I found this: https://en.bitcoin.it/wiki/BIP_0341#cite_note-7

Even if you don't add it to code it'll live in this PR.

Reply:
I'll add it in the follow-up